src/components/productdisplayitem.main.jsx

```diff
@@ -446,6 +477,9 @@ class ProductDisplayItemMain extends React.Component {
                         </option>
                       </select>
                     </div>
+                    {
+                      (isLoading) ? (<div className="miniLoader" />) : ''
+                    }
```

Comment:
I moved this there because I believe it looks better after I added the cart item modifiers